### PR TITLE
Add pretty error messages

### PR DESCRIPTION
**Pull request by hulthe**

### CODE REVIEW

File: src/ast/mod.rs [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
-    Value(Value<'a>),
-    Equals(Box<Expr<'a>>, Box<Expr<'a>>),
-    NotEquals(Box<Expr<'a>>, Box<Expr<'a>>),
-    LessEquals(Box<Expr<'a>>, Box<Expr<'a>>),
-    LessThan(Box<Expr<'a>>, Box<Expr<'a>>),
-    GreaterThan(Box<Expr<'a>>, Box<Expr<'a>>),
-    GreaterEquals(Box<Expr<'a>>, Box<Expr<'a>>),
-    And(Box<Expr<'a>>, Box<Expr<'a>>),
-    Or(Box<Expr<'a>>, Box<Expr<'a>>),
+    #[serde(borrow)]
```
Comment (ejohnsson6) [inline src/ast/mod.rs]: Any reason for renaming these?

Comment (hulthe) [reply src/ast/mod.rs]: It looks better when it all lines up :)

Comment (Baloo1) [reply src/ast/mod.rs]: Could it be confusing to have some of the shorter names like LT? I am not sure if some of the shorter names are too non-descriptive. What do you think?

Comment (feffes) [reply src/ast/mod.rs]: I think it's hard to misinterpret given the context.

Comment (hulthe) [reply src/ast/mod.rs]: Would three letters be acceptable :wink: (e.g. LTh, GEq, NEq, Eql)?

Comment (Baloo1) [reply src/ast/mod.rs]: I think that would be an improvement yes!

**Review by ejohnsson6 — APPROVED** [review]

**Review by Baloo1 — APPROVED** [review]
I think this look good, @Swexbe what do you think?

**Pull request merged**